main.go

```diff
@@ -6,6 +6,7 @@ package main
 import (
 	goflag "flag"
 	"fmt"
+	"github.com/projectcapsule/capsule/pkg/webhook/ownerreference"
```

Comment:
Project imports are always part of the first block. Import ordering:

1. Std-Libs
2. 3-Party
3. Imports from this project